### ISSUE TITLE
Bug: mongodb adapter was not stopped when broker stop

### ISSUE THREAD

**Issue by zhaoyao91** [moleculerjs/moleculer-db#24]
When I stop the broker, the mongodb adapter was not stopped correctly, so although the test run successfully, it doesn't exit. I have to add the workaround code to my service:
```
  async stopped() {
    await this.adapter.client.close(true)
  }
```

**Comment by icebob** [comment]
This `close` is called, but the Promise is not returned: https://github.com/moleculerjs/moleculer-db/blob/master/packages/moleculer-db-adapter-mongo/src/index.js#L81

I think it is the problem, right?

**Comment by zhaoyao91** [comment]
yeah, this is one reason. there is some tricky point for mongo client.close, you should pass true to it to force close, otherwise it will hang there.
